.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,24 @@
+name: golangci-lint
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - prod
+      - dev
+  pull_request:
+    branches:
+      - master
+      - prod
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
```

Comment:
same, pin to full commit sha for golangci-lint-action

Reply:
fixed thanks